Exercises/4-count-types.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
-const countTypesInArray = null;
+/* const arr = [true, 3, '4', false, { n: 3 }, function (x) { return x + 1; },
+  { name: 'Marc' }, 'Marc', 3.11, [1, 2, 4], 5];*/
+
+const countTypesInArray = arr => {
+  const result = {};
+  for (const item of arr) {
+    const temp = typeof(item);
```

Comment:
```suggestion
    const temp = typeof item;
```

Reply:
Fix naming: `temp` is not ok

Reply:
So JS allows to use both syntax: typeof var and typeof(var)? Cool

Reply:
`typeof` is not a function, it is operator, so preffered syntax is `typeof <identifier>` but `typeof(<identifier>)` is not a bug but it's a style issue.